icx.spec

```diff
@@ -0,0 +1,14 @@
+### RPM external icx 2022
```

Comment:
@gartung , this is not part of cmssw-tool-conf so it is not going to build. I suggest to add it in https://github.com/cms-sw/cmsdist/blob/4de64f2d0118131384741ec77ed0977248bf49ee/cmssw-tool-conf.spec#L164 section (i.e. only for x86_64). So move icc to be in x86_64 section only